prov/ucx/src/ucx_init.c

```diff
@@ -332,7 +332,8 @@ static int ucx_getinfo(uint32_t version, const char *node,
 		    hints->addr_format == FI_ADDR_UCX ||
 		    hints->addr_format == FI_FORMAT_UNSPEC) {
 			ucx_info.addr_format = FI_ADDR_UCX;
-		} else if (hints->addr_format <= FI_SOCKADDR_IN) {
+		} else if (hints->addr_format <= FI_SOCKADDR_IN ||
+			   hints->addr_format == FI_SOCKADDR_IP) {
```

Comment:
Use is_ipv4addr() ?

Reply:
That requires moving the definitions to `ofi.h`. Is it worth the trouble?

Reply:
I pushed an update that addressed all comments except this one. If you think we should move the help functions to ofi.h I can do that, otherwise I will leave it as it is.